### PR TITLE
Add support to generate metrics about testing coverage in schema CLI and upload to Codecov

### PR DESCRIPTION
**Pull request by maniarathi**
### Additions

- Add Codecov badge to the top level repo README.
- Upload coverage report to Codecov in push_test Github Action.

### Modifications

- Alphabetize `requirements.txt`
- Run `coverage` instead of `pytest` to execute unit tests so that coverage stats are generated.

### CODE REVIEW

File: cellxgene_schema_cli/requirements.txt [hunk 1]
```diff
@@ -1,9 +1,10 @@
-wheel==0.37.0
+anndata==0.7.6
 click==8.0.1
+coverage==6.3.2
```
Comment (seve) [inline cellxgene_schema_cli/requirements.txt:3]: Only substantial change in file I believe

File: Makefile [hunk 2]
```diff
@@ -9,7 +9,7 @@ install:
 	cd cellxgene_schema_cli/ && pip install -r requirements.txt && pip install .
 
 unit-test:
-	cd cellxgene_schema_cli && pytest
+	cd cellxgene_schema_cli && coverage run --source=cellxgene_schema -m unittest discover --start-directory=tests/ --verbose && coverage xml
```
Comment (seve) [inline Makefile:12]: Do we not need a CODECOV token like we do in dataportal?

Comment (maniarathi) [reply Makefile:12]: Nope. It is a public repository so we don't need a token.

File: cellxgene_schema_cli/requirements.txt (hunk 1 above)
Comment (seve) [reply cellxgene_schema_cli/requirements.txt:3]: was just commenting so Pablo didn't have to manually diff too!

**Pull request merged**